test/integration/go.mod

```diff
@@ -4,11 +4,13 @@ go 1.12
 
 require (
 	cloud.google.com/go v0.38.0 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.8.5 // indirect
```

Comment:
Whups, not sure why the modules file got changed. I'll revert this here in a few.